devel/fortio/cmd/fortio/main.go

```diff
@@ -87,13 +83,12 @@ func main() {
 		compressionFlag = flag.Bool("compression", false, "Enable http compression")
 		goMaxProcsFlag  = flag.Int("gomaxprocs", 0, "Setting for runtime.GOMAXPROCS, <1 doesn't change the default")
 		profileFlag     = flag.String("profile", "", "write .cpu and .mem profiles to file")
-		headersFlags    flagList
+		httpClient      = flag.Int("http", 1,
```

Comment:
this one is only in the main, so it's not really an api design but just how to expediently select which client and option to use, I do agree strings would be cleaner and an auto mapping from string<->enum like I implemented in logger.go is nicer but it's also quite a lot more code - I will make some change, thanks!